docs/MPURegionsUsageGuide.md

```diff
@@ -100,8 +104,22 @@
 				- [ARMv7-M MPU functions](../os/arch/arm/src/armv7-m/mpu.h)
 				- [ARMv7-R MPU functions](../os/arch/arm/src/armv7-r/mpu.h)
 				- [ARMv8-M MPU functions](../os/arch/arm/src/armv8-m/mpu.h)
-		- For repeated/customized configuration of a MPU region, a two-step process should be applied using below functions:
-			1. In the first step, we obtain the mpu register values and store them in an array using below function:
+		- For repeated/customized configuration of a MPU region, a three-step process should be applied using below functions:
+			1. In the first step, get the region number to be configured using the below API:
+			```c
+			uint8_t nregion = mpu_get_nregion_info(MPU_REGION_xx);
+			```
+			Similarly, region numbers for other MPU usages can be obtained using the API for following enums:
```

Comment:
Here "region numbers for other MPU usages" is not clear. 
Let us change this line as "Following enum provides possible mpu types to use in above api."